docs/Getting Started/Integration Basics/index.md

```diff
@@ -1236,3 +1236,127 @@ The `SDLSessionBean` should be inside a Java package other than the default pack
 * Create Manifest. Apply + OK.
 * Run Build -> Build Artifacts to get a .war file in the /out folder.
 !@
+
+@![javascript]
+The type of app you can make will depend on the build library you select. If you are using the Node.js build, your app can run as a WebSocket server or as a TCP client. If you are using the vanilla JavaScript build, your app can run as a WebSocket client. This guide will cover topics that apply to both Node.js and plain JS library builds. Specific code snippets for each environment will be shown when applicable.
+
+## Basic Configuration
+In order to correctly connect to an SDL enabled head unit, developers need to create a configuration object to pass into an instance of the `SdlManager` class and then start it. This configuration object is the `AppConfig`, which requires a `LifecycleConfig` object. `LifecycleConfig` contains the majority of the settings a developer would need to set in order for the app to be operable. At the very least, you will want to use the following methods to get started: `setAppId`, `setAppName`, `setLanguageDesired`, `setAppTypes`, and `setTransportConfig`. These configuration objects support method chaining, which allow you to create the following example configuration object:
+
+```js
+const lifecycleConfig = new SDL.manager.LifecycleConfig()
+    .setAppId('hello-js')
+    .setAppName('Hello JS')
+    .setLanguageDesired(SDL.rpc.enums.Language.EN_US)
+    .setAppTypes([
+        SDL.rpc.enums.AppHMIType.DEFAULT,
+    ]);
+```
+
+For WebEngine apps, most of this configuration will happen in the `manifest.js` file. The `LifecycleConfig` class has a `loadManifest` method that takes in a manifest file and sets up the configuration itself.
+
+## Transport Configuration
+The transport configuration will depend on the environment you're using. For instance, you can make a TCP connection with Node.js, but not with vanilla JS. Setting up the transport correctly will be covered in the next section. See below for example transport configurations.
```

Comment:
```suggestion
The transport configuration will depend on the environment you're using. For example, you can make a TCP connection with Node.js, but not with vanilla JS. Setting up the transport correctly will be covered in the next section. See below for example transport configurations.
```

Reply:
> Setting up the transport correctly will be covered in the next section

Do you mean another guide? If so, please link it. If not, you can remove this sentence.